console/services/market_app/market_app.py

```diff
@@ -95,7 +103,12 @@ def ensure_volume_deps(original_app: OriginalApp, new_deps):
         """
         # 保留 app_id 和 upgrade_group_id 都不同的依赖关系
         # component_ids 是相同 app_id 和 upgrade_group_id 下的组件, 所以 dep_service_id 不属于 component_ids 的依赖关系属于'情况2'
+        if is_upgrade_one:
```

Comment:
Upgrade one component. If the dependency of the component has changed with other components (existing in the template and installed), then update it.

Reply:
fixed